pages/common/unlink.md

```diff
@@ -0,0 +1,7 @@
+# unlink
+
+> Remove a file from a filesystem.
+
+- Remove the specified file if it is the last link:
```

Comment:
This makes the user wonder what happens if it is _not_ the last link. I find the man page description clear enough. How about - "Delete a name from the file system. (If the name was the last link, the file is removed too)".